src/modules/projects/components/tables/ProjectClientEnrollmentsTable.tsx

```diff
@@ -40,65 +45,135 @@ const isHouseholdWithStaff = (
+  HouseholdWithStaffAssignmentsFragment,
+  EnrollmentQueryVariables
+> = {
+  header: 'Assigned Staff',
+  optional: {
+    defaultHidden: true,
+    queryVariableField: 'includeStaffAssignment',
+  },
+  key: 'assigned_staff',
+  render: renderHouseholdAssignedStaff,
+};
+
 export const ENROLLMENT_COLUMNS: {
-  [key: string]: ColumnDef<
-    | ClientEnrollmentFieldsFragment
+  [key: string]: GenericTableWithDataColumnDef<
+    | ClientEnrollmentTableFields
     | ProjectEnrollmentQueryEnrollmentFieldsFragment
+    | ClientEnrollmentFieldsFragment,
+    EnrollmentQueryVariables
   >;
 } = {
   entryDate: {
     header: 'Entry Date',
+    key: 'entryDate',
     render: (e) => (
       <DateWithRelativeTooltip dateString={e.entryDate} preciseTime={false} />
     ),
   },
   exitDate: {
     header: 'Exit Date',
+    key: 'exitDate',
+    optional: {
+      defaultHidden: true,
+      // queryVariableField not provided here, since we need to fetch exitDate anyway in order to show the status
+      // and correctly aria-label the row action
+    },
     render: (e) => {
-      if (e.exitDate)
+      if (e.exitDate) {
         return (
           <DateWithRelativeTooltip
             dateString={e.exitDate}
             preciseTime={false}
           />
         );
+      }
     },
   },
   enrollmentStatus: {
     header: 'Status',
+    key: 'status',
     render: (e) => <EnrollmentStatus enrollment={e} />,
   },
-};
-
-export const ASSIGNED_STAFF_COL = {
-  header: 'Assigned Staff',
-  optional: true,
-  defaultHidden: true,
-  key: 'assigned_staff',
-  render: (hh: HouseholdWithStaffAssignmentsFragment) => {
-    if (!hh.staffAssignments?.nodes.length) return;
-
-    const allNames = hh.staffAssignments.nodes.map(
-      (staffAssignment) => staffAssignment.user.name
-    );
-
-    const first = allNames[0];
-    const rest = allNames.slice(1);
-
-    return (
-      <Box aria-label={allNames.join(', ')}>
-        {first}{' '}
-        {rest.length > 0 && (
-          <Tooltip arrow title={rest.join(', ')}>
-            <Chip
-              sx={{ mb: 0.5 }}
-              size='small'
-              label={`+${rest.length} more`}
-            />
-          </Tooltip>
-        )}
-      </Box>
-    );
+  moveInDate: {
+    header: 'Move-in Date',
+    key: 'moveInDate',
+    optional: {
+      defaultHidden: true,
+      queryVariableField: 'includeMoveInDate',
+    },
+    render: (e) => {
+      if (e.moveInDate) {
+        return (
+          <DateWithRelativeTooltip
+            dateString={e.moveInDate}
+            preciseTime={false}
+          />
+        );
+      }
+    },
+  },
+  lastContactDate: {
+    header: 'Last Contact Date',
+    key: 'lastContactDate',
+    optional: {
+      defaultHidden: true,
+      queryVariableField: 'includeLastContact',
+    },
+    render: (e) => {
+      if ('lastContact' in e && e.lastContact) {
+        return (
+          <Box flexDirection='row'>
```

Comment:
This is probably a design question, but just throwing it out there: in what situation does it make sense to wrap the cell content, vs not wrap the cell content and rely on horizontal scroll?
We did invest time into the horizontal scroll and sticky columns, so maybe it makes sense to remove the wrapping in some instances. Here `whiteSpace='nowrap'` would do that, imo it makes it easier to parse, and we know the content will never be super long (unlike client name).